tools/flaky-test-reporter/jsonreport/jsonreport.go

```diff
@@ -40,6 +40,10 @@ type Report struct {
 	Flaky []string `json:"flaky"`
 }
 
+func Initialize(credentials string) error {
+	return prow.Initialize(credentials)
```

Comment:
suggest also creates a struct for handling everything jsonreport related, so the caller doesn't receive `prow` struct when call `Initialize`

Reply:
I don't believe there is a struct or a client in this case, unlike other setup functions we use. prow.Initialize() is just a wrapper for gcs.Authenticate(), and all structs are private within that package.